### PR TITLE
SwG

### PR DESCRIPTION
**Pull request by remybach**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,9 +1,29 @@
 'use strict';
 
 const herokuCluster = require('@quarterto/heroku-cluster');
+const https = require('https');
+const denodeify = require('denodeify');
```
Comment (quarterto) [inline app.js]: we already have `@quarterto/promisify` as a dependency which is the same thing but written by me 😉

File: app.js [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+const isDevelopment = app.get('env') === 'development';
+
+if(isDevelopment) {
+	const readFile = denodeify(fs.readFile);
+	const keyFile = readFile(path.resolve(__dirname, 'key.pem'));
+	const certFile = readFile(path.resolve(__dirname, 'cert.pem'));
+	Promise.all([keyFile, certFile])
+		.then(([key, cert]) => {
+			https.createServer({ key, cert }, app)
+				.listen(5050, () => console.log('App listening on port 5050 (https) in development'));
```
Comment (quarterto) [inline app.js]: ft-app uses `httpolyglot` to run http and https on the same port, which works really well http://git.svc.ft.com/projects/FTA/repos/ft-app/browse/server/index.js

Comment (remybach) [reply app.js]: Ooh, this is much neater!

File: package.json [hunk 3]
```diff
@@ -121,6 +122,7 @@
     "eslint": "^3.12.2",
     "eslint-config-airbnb-base": "^11.0.0",
     "eslint-plugin-import": "^2.2.0",
+    "https": "^1.0.0",
```
Comment (quarterto) [inline package.json]: this looks unnecessary?

File: views/article.html [hunk 4]
```diff
@@ -49,20 +47,18 @@ <h2 class="article-standfirst">{{standfirst}}</h2>
 		</div>
 	</header>
 
-	<div amp-access="access OR {{showEverything}}" amp-access-hide>
-		<div class="article-body" itemprop="articleBody">
-			{{{htmlBody}}}
+	<div class="article-body" itemprop="articleBody">
```
Comment (quarterto) [inline views/article.html]: does this need a `subscriptions-display` or a `subscriptions-section` attribute? how does it get hidden?

Comment (quarterto) [reply views/article.html]: ditto for header & comments

Comment (remybach) [reply views/article.html]: The `subscriptions-section="content"` bit here wraps around all of that and is what `amp-subscriptions` displays when access is granted: https://github.com/Financial-Times/google-amp/pull/279/files#diff-febb4d6e59192de3d32fc452273edc4fR162

Comment (quarterto) [reply views/article.html]: ah that makes sense

Comment (quarterto) [reply views/article.html]: ah that makes sense

**Pull request merged**